extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/JacksonBuildTimeConfig.java

```diff
@@ -1,9 +1,10 @@
 package io.quarkus.jackson.runtime;
 
 import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
 
-@ConfigRoot
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
```

Comment:
Didn't we have an issue with this and had to revert in https://github.com/quarkusio/quarkus/pull/14644 ?

Reply:
Hmmm, so that's where this feeling of deja vu was coming from :).

Reply:
Yeah... There is some kinda mess with the kubernetes client that https://github.com/quarkusio/quarkus/pull/14609 hopefully can address

Reply:
@geoand I adjusted the PR!